### PR TITLE
Bottom sheet copy date

### PR DESCRIPTION
**Pull request by MertzAndreas**
## Description

Migrated copy date to BottomSheet and made some 'interesting' performance improvements.

## Type of PR

- Bugfix
- Refactor

## Related Issues/Tickets

None

## Checklist

- [X] Completed the user story described by the issue (if applicable)
- [X] Code follows the code guidelines
- [X] Tests have been added/updated (if applicable)
- [X] Documentation has been updated (if applicable)

### CODE REVIEW

File: components/SwipeableList/SwipeableList.tsx [hunk 1]
```diff
@@ -70,11 +75,20 @@ const SwipeableList = <T,>({
...
   const handleLayout = (event: LayoutChangeEvent) => {
     const { height } = event.nativeEvent.layout;
-    if (height !== itemDimensions) setItemHeight(height);
+
+    /*
+    I admit this is iffy, but it's the best I could come up with.
+    I noticed a big performance decrease with larger lists. 
+    After some testing, I narrowed the issue to a small unnoticeeable difference in height between items.
+    This caused a rerender for every item in the list, which was very slow. 🤮🎪🤡🐡
+    */
```
Comment (Tbjerregaard) [inline components/SwipeableList/SwipeableList.tsx:88]: 🥇

File: components/weekoverview_components/activity_components/ActivityItem.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const ActivityItem: React.FC<ActivityItemProps> = ({ time, isCompleted, setImage
         {data ? (
           <Pressable onPress={() => handleImagePress(data)}>
             <Image
-              source={{ uri: data }}
+              source={{ uri: data, cache: "force-cache" }}
```
Comment (Tbjerregaard) [inline components/weekoverview_components/activity_components/ActivityItem.tsx:52]: How much % performance increase is this? I think it's around 0% after reading docs, but I might be wrong

File: components/weekoverview_components/activity_components/ActivityItemList.tsx [hunk 3]
```diff
@@ -30,6 +30,44 @@ const ActivityItemList = () => {
...
+  const handleDeleteActivity = useCallback(
+    async (id: number) => {
+      await useDeleteActivity.mutateAsync(id);
+    },
+    [useDeleteActivity]
+  );
+
+  const handleEditTask = useCallback((item: ActivityDTO) => {
+    router.push({
+      pathname: "./editactivity",
```
Comment (Tbjerregaard) [inline components/weekoverview_components/activity_components/ActivityItemList.tsx:54]: please specify the full path for consistency

**Review by Tbjerregaard — CHANGES_REQUESTED** [review]

File: components/weekoverview_components/activity_components/ActivityItem.tsx (hunk 2 above)
Comment (MertzAndreas) [reply components/weekoverview_components/activity_components/ActivityItem.tsx:52]: anything over 0 is good 💯

Comment (MertzAndreas) [reply components/weekoverview_components/activity_components/ActivityItem.tsx:52]: Why u bully me

**Review by Tbjerregaard — APPROVED** [review]

**Review by MaksimHamko — APPROVED** [review]
Looks good

**Review by Tbjerregaard — APPROVED** [review]

**Pull request merged**